### DISCUSSION TITLE
Theia IDE Desktop, V 1.51.0. Can't select "Go to Definition" function item at popped up sub-menu.

### DISCUSSION THREAD

**Discussion by dnsjong** [eclipse-theia/theia#13921] [Improvements] [answered]
When we attempt to "Go to Definition" with right clicked mouse at editing [keyword], Theia IDE will pop up sub-menu
for selection where we can't pick up menu-item at top fist line of "Go to Definition", except at "FULL SCREEN" mode.

Dennis

**Comment by msujew** [comment, marked as answer]
Hey @dnsjong,

this sounds like a bug report, right? Can you create a bug report issue with reproduction steps (and ideally some screenshots)? Much appreciated!

**Reply by dnsjong** [reply]
Yes, thanks! 
As I checked to "issue" list, it was reported on bug #13081
